CHANGELOG.md

```diff
@@ -15,6 +15,12 @@
         * Change `PayPalLineItemKind` to an enum
         * Rename `PayPalLineItemKind.KIND_CREDIT` to `CREDIT`
         * Rename `PayPalLineItemKind.KIND_DEBIT` to `DEBIT`
+        * Update `ThreeDSSecureInfo.isLiabilityShifted()` to `ThreeDSSecureInfo.getLiabilityShifted()`
+        * Update `ThreeDSSecureInfo.isLiabilityShiftPossible()` to `ThreeDSSecureInfo.getLiabilityShiftPossible()`
+        * Update `ThreeDSSecureInfo.wasVerified()` to `ThreeDSSecureInfo.getWasVerified()`
```

Comment:
I might have missed the context on this - are we changing these for better Kotlin interop or consistency across modules?

Reply:
Unfortunately, to utilize the `@Parcelize` annotation, all properties must be declared in the primary constructor. We'd need to add a specific getter for java integrations.

Here's a related thread on a previous PR: https://github.com/braintree/braintree_android/pull/1088#discussion_r1698935948

Reply:
Got it - thanks 👍